Update Improved NPC Faces for OCOv2

YokiHotsumi opened the pull request:
*Add load order
- OCO Cleaned - Vanilla Hair.esp
Must be load before
- Improved NPC Faces for OCOv2.esp
- OCO Nord and Redguards Eyes Fix.esp

Improved compatibility of "Improved NPC Faces" when using "OCO Cleaned - Vanilla Hair".

YokiHotsumi committed:
Update OCO Cleaned - Vanilla Hair

*Add load order
- OCO Cleaned - Vanilla Hair.esp
Must be load before
- Improved NPC Faces for OCOv2.esp
- OCO Nord and Redguards Eyes Fix.esp

Improved compatibility of "Improved NPC Faces" when using "OCO Cleaned - Vanilla Hair".

sibir-ine commented on masterlist.yaml:
```diff
@@ -4637,6 +4637,9 @@ plugins:
     url:
       - link: 'https://www.nexusmods.com/oblivion/mods/51324/'
         name: 'Oblivion Character Overhaul Cleaned - OCO Cleaned'
+    before:
+      - 'Improved NPC Faces for OCOv2.esp'
+      - 'OCO Nord and Redguards Eyes Fix.esp'
```
Just to be sure, I'm assuming `Improved NPC Faces for OCOv2.esp` is from [here](https://www.nexusmods.com/oblivion/mods/51971) & `OCO Nord and Redguards Eyes Fix.esp` is from [here](https://www.nexusmods.com/oblivion/mods/52041)? From looking at the plugins in xEdit, `OCO Nord and Redguards Eyes Fix.esp` & `OCO Cleaned - Vanilla Hair.esp` don't conflict (at least not in any way that would be affected by load order), so a rule isn't necessary for them.

A rule can be added for `Improved NPC Faces for OCOv2.esp`. However, there is no `before` key for enumerating plugins that a plugin should load before, only an `after` key for plugins that a plugin should load after. You'd therefore want to add the following line to the [existing plugin](https://github.com/loot/oblivion/blob/fbf6c5419e85e05bb955ceb4e8112e0dc1781ed3/masterlist.yaml#L4405-L4409) for `Improved NPC Faces for OCOv2.esp`, specifically just above the `inc` key:

```yaml
    after: [ 'OCO Cleaned - Vanilla Hair.esp' ]
```

The plugin should look like this:

```yaml
  - name: 'Improved NPC Faces for OCOv2.esp'
    after: [ 'OCO Cleaned - Vanilla Hair.esp' ]
    inc: [ 'GreyPrince.esp' ]
    clean:
      - crc: 0x76E1B1BF
        util: 'TES4Edit v4.0.4b'
```

YokiHotsumi replied:
Hello,

Yep that's correct, they are from like you said. The fact is, OCO Vanilla Hair overrides the changes of similars npcs on both plugins, if he load after Improved NPC Faces.
I understand, so i can make the change like you said, a rule after must be good for the plugin Improved NPC Faces.
I'am sorry for my english, i'm a french user.

sibir-ine replied:
No worries, feel free to make the changes in this PR or just close it & open a new one, whichever is easier for you.

Also, if you're interested in contributing French translations for LOOT, we have https://github.com/loot/loot.github.io/issues/77 open to provide a general overview with any links you'd need. Our French translation is pretty much complete (there are untranslated one-off messages throughout the masterlists, but most of them will likely be removed or replaced with translated messages), but we always welcome revisions & translations in the future for any strings that get added.

YokiHotsumi replied:
Thanks for your help. I am interested in this french branch. I'll look into it, and see if I can contribute.

sibir-ine replied:
Thanks! We'd appreciate it.

YokiHotsumi committed:
Update Improved NPC Faces for OCOv2

*Add load order

- Improved NPC Faces for OCOv2.esp
Must be load after
- OCO Cleaned - Vanilla Hair.esp

Improved compatibility of "Improved NPC Faces" when using "OCO Cleaned - Vanilla Hair".

sibir-ine reviewed (approved):
Thanks!
